test/functional/test_framework/test_framework.py

```diff
@@ -204,11 +213,13 @@ def setup(self):
             "bitcoin-cli" + config["environment"]["EXEEXT"],
         )
         self.options.bitcoind = os.getenv("BITCOIND", default=fname_bitcoind)
-        self.options.bitcoincli = os.getenv("BITCOINCLI", default=fname_bitcoincli)
+        self.options.bitcoincli = os.getenv(
+            "BITCOINCLI", default=fname_bitcoincli)
 
         os.environ['PATH'] = os.pathsep.join([
             os.path.join(config['environment']['BUILDDIR'], 'src'),
-            os.path.join(config['environment']['BUILDDIR'], 'src', 'qt'), os.environ['PATH']
+            os.path.join(config['environment']['BUILDDIR'],
+                         'src', 'qt'), os.environ['PATH']
```

Comment:
I am not sure it this is actually more readable (we've generally been using more a limit of 120 instead of 80).
And taking into account the large number of conflicts due to the pep-8 change, maybe leave it out for now?

Reply:
Reverted